web/src/main/java/com/synopsys/integration/alert/web/documentation/SwaggerConfiguration.java

```diff
@@ -37,4 +38,9 @@ public OpenAPI springShopOpenAPI() {
                 .version("preview"));
     }
 
+    @Bean
```

Comment:
Create the bean that customizes the operation IDs.  When the Swagger UI is accessed initially, the customizer bean is invoked to update the operation IDs.